closed/OpenJ9.gmk

```diff
@@ -32,6 +32,11 @@ ifeq (,$(BUILD_ID))
   BUILD_ID := 000000
 endif
 
+OPENJDK_SHA := $(shell git -C $(TOPDIR) rev-parse --short HEAD)
+ifeq (,$(OPENJDK_SHA))
+  $(error Could not determine OpenJDK SHA)
```

Comment:
When could this occur?

Reply:
It shouldn't happen in a normal workflow, but if you had just the working directory, with no `.git` directory it would fail.
The pattern here follows that for `OPENJ9_SHA` and `OPENJ9OMR_SHA`.